webapi/src/test/scala/org/knora/webapi/messages/admin/responder/permissionsmessages/PermissionsMessagesADMSpec.scala

```diff
@@ -57,14 +57,15 @@ class PermissionsMessagesADMSpec extends CoreSpec() {
     }
 
     "return 'BadRequest' if the supplied permission IRI for AdministrativePermissionForIriGetRequestADM is not valid" in {
+      val permissionIri = "invalid-permission-IRI"
       val caught = intercept[BadRequestException](
         AdministrativePermissionForIriGetRequestADM(
-          administrativePermissionIri = "invalid-permission-IRI",
+          administrativePermissionIri = permissionIri,
           requestingUser = SharedTestDataADM.imagesUser01,
           apiRequestID = UUID.randomUUID()
         )
       )
-      assert(caught.getMessage === "Invalid permission IRI")
+      assert(caught.getMessage === s"Invalid permission IRI $permissionIri is given.")
     }
 
     "return 'ForbiddenException' if the user requesting AdministrativePermissionForIriGetRequestADM is not SystemAdmin" in {
```

Comment:
Test for not ProjectAdmin is missing.

Reply:
the title of the test is wrong, it checks for the project admin.